content/docs/tasks/traffic-management/egress-gateway.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Direct Egress Traffic through Egress Gateway
+description: Describes how to configure Istio to traffic to external services through a dedicated service
+weight: 43
+---
+
+> This task uses the new [v1alpha3 traffic management API](/blog/2018/v1alpha3-routing/). The old API has been deprecated and will be removed in the next Istio release. If you need to use the old version, follow the docs [here](https://archive.istio.io/v0.7/docs/tasks/traffic-management/). Note that this task introduces a new concept, namely Egress Gateway, that was not present in previous Istio versions.
+
+The [Control Egress Traffic](/docs/tasks/traffic-management/egress/) task demonstrates how external (outside the Kubernetes cluster) HTTP and HTTPS services can be accessed from applications inside the mesh. A quick reminder: by default, Istio-enabled applications are unable to access URLs outside the cluster. To enable such access, a [ServiceEntry](/docs/reference/config/istio.networking.v1alpha3/#ServiceEntry) for the external service must be defined, or, alternatively, [direct access to external services](/docs/tasks/traffic-management/egress/#calling-external-services-directly) must be configured.
+
+The [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task demonstrates how to allow the applications to send HTTP requests to external servers that require HTTPS.
+
+This task describes how to configure Istio to direct the egress traffic through a dedicated service called _Egress Gateway_. We extend the use case described in the [TLS Origination for Egress Traffic](/docs/tasks/traffic-management/egress-tls-origination/) task.
+
+## Use case
+
+Consider an organization that has strict security requirements. According to these requirements all the traffic that leaves the service mesh must flow through a set of dedicated nodes. These nodes will run on dedicated machines, separately from the rest of the nodes used for running applications in the cluster. The special nodes will serve for policy enforcement on the egress traffic and will be monitored more thoroughly than the rest of the nodes.
+
+Istio 0.8 introduced the concept of [ingress and egress gateways](/docs/reference/config/istio.networking.v1alpha3/#Gateway). Ingress gateways allow defining entrance points into the service mesh so all the incoming traffic flows through these entrance points. _Egress gateway_ is a symmetrical concept, it defines exit points for the mesh. An egress gateway allows Istio features, for example, monitoring and route rules, to be applied to traffic exiting the mesh.
+
+Additional use case is a cluster where the application nodes do not have public IPs. Defining an egress gateway, directing all the egress traffic through it and allocating public IPs to the egress gateway nodes allows the application nodes access external services in a controlled way.
```

Comment:
I don't understand this. What application nodes are you referring to? The services/pods in the mesh? What does their lack of public IPs have to do with accessing external services?

Reply:
Rephrased: 

> An additional security measure is configuring the network in such a way that the application nodes are unable to access the Internet without directing the egress traffic through the gateway where it will be monitored and controlled. One example of such network configuration is allocating public IPs exclusively to the gateways.

Reply:
Another possible security measure involves configuring ...